crates/flox/src/commands/mod.rs

```diff
@@ -469,7 +471,46 @@ impl EnvironmentSelect {
                             if activated == current_dir {
                                 Self::open_env_pointer(flox, &current_dir, current_dir_pointer)
                             } else {
-                                todo!("needs design");
+                                let activated_pointer = EnvironmentPointer::open(&activated)?;
+                                let message = "Do you want to install to the current directory's flox environment or the current active flox environment?";
```

Comment:
Should be a message non-specific to the action we are performing (e.g. "install" here) or we have to push in the action from all use sides

Reply:
Oh yeah, I guess I'll pass in the command name

Reply:
Fixed